src/gui/newwizard/pages/serverurlsetupwizardpage.cpp

```diff
@@ -17,22 +17,13 @@ ServerUrlSetupWizardPage::ServerUrlSetupWizardPage(const QUrl &serverUrl)
 
     _ui->urlLineEdit->setText(serverUrl.toString());
 
-    // first, we declare this object as an event filter
-    // this then allows us to make the controller the event filter of the current page
-    _ui->urlLineEdit->installEventFilter(this);
```

Comment:
This change appears unrelated to the rest.

Reply:
Yes but as the class is not implementing an event filer....?

Reply:
I'm not a pro at this, but if I remember this correctly, I had to install the window as an event filter manually, otherwise I would not have received the events upstream. If this is no longer necessary (or maybe never was), great!